game/models.py

```diff
@@ -172,17 +177,18 @@ def make_standard_deck():
 class Dungeon:
     """Handle deck, room and player creation and interaction"""
 
-    def __init__(self, cards=make_standard_deck(), seed=None):
+    def __init__(self, cards=None, seed=None):
```

Comment:
Like this idea of passing a set of cards to the dungeon. I guess you could recreate a certain point in a dungeon from a game history and see if a game was winnable from a certain point.

Which makes me think we should store the state of the deck in the game history, in addition to the console output.